scripts/variantstore/wdl/extract/generate_FOFNs_for_import.py

```diff
@@ -1,22 +1,32 @@
-import os
 import argparse
+import time
 
 from terra_notebook_utils import table
-from terra_notebook_utils import workspace
 
-import time
 
-# just keep these around as constants for now
-vcf_files_name = "vcf_files.txt"
-vcf_index_files_name = "vcf_index_files.txt"
-sample_names_file_name = "sample_names.txt"
-error_file_name = "errors.txt"
+def get_entities_in_set(data_table_name, sample_set_name):
+    set_of_entities = set()
+    sample_set_list = list(table.list_rows(f"{data_table_name}_set"))
+    for sample_set in sample_set_list:
+        if sample_set.name == sample_set_name: ## else it's some other sample set and we dont care about it
+            # for sample in sample_set.attributes['samples']: ## TODO why is this samples in Terra? Is this some Terra hardcoding?!?!
```

Comment:
```suggestion
```

Reply:
nope!

Reply:
I was wrong!

Reply:
ugh this is so ugly
for sample in sample_set.attributes[f"{data_table_name}s"]: